drivers/performance_measurement/app/controllers/performance_measurement/warehouse_reports/reports_controller.rb

```diff
@@ -95,6 +95,16 @@ def clients
       end
     end
 
+    def equity_analysis
+      @equity_filters = params[:equity_filters]
+      @analysis_builder = PerformanceMeasurement::EquityReportBuilder.new(@equity_filters)
+      if @equity_filters.present?
+        # if there are filters set errors
+        flash[:error] = @analysis_builder.valid? ? '' : 'There was an error building the equity analysis.'
```

Comment:
This doesn't work (flash message). Do you have a pattern for showing errors? We can remove this but we need to call `valid?` on the builder, I think. That way errors will show up on the form fields.